langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaRequestLoggingInterceptor.java

```diff
@@ -0,0 +1,43 @@
+package dev.langchain4j.model.ollama;
+
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+import okio.Buffer;
+
+import java.io.IOException;
+
+@Slf4j
+class OllamaRequestLoggingInterceptor implements Interceptor {
+
+    @Override
+    public Response intercept(Chain chain) throws IOException {
+        Request request = chain.request();
+        this.log(request);
+        return chain.proceed(request);
+    }
+
+    private void log(Request request) {
+        try {
+            log.debug("Request:\n- method: {}\n- url: {}\n- headers: {}\n- body: {}",
+                    request.method(), request.url(), request.headers(), getBody(request));
```

Comment:
Headers (that might contain api keys) are not sanitized, please see `AnthropicRequestLoggingInterceptor` as example

Reply:
ok